browser/ui/webui/settings/brave_search_engines_handler.h

```diff
@@ -24,6 +28,9 @@ class BraveSearchEnginesHandler : public SearchEnginesHandler {
   BraveSearchEnginesHandler& operator=(const BraveSearchEnginesHandler&) =
       delete;
 
+  // BraveOriginStateObserver:
+  void OnBraveOriginStatusChanged() override;
```

Comment:
Just to make sure I didn't miss something- nothing is calling this yet, right? I see the `AddObserver` / `RemoveObserver` definitions - but nothing calling them and then nothing calling `OnBraveOriginStatusChanged`.  

This will be handy once we get into the receipt logic (person goes from unpaid / paid)!

Reply:
yes, that's right